qtc_packaging/debian_fremantle/changelog

```diff
@@ -1,3 +1,8 @@
+kodimote (2.0.2-fremantle0) unstable; urgency=low
+  * transition from xbmcremote to kodimote
+
+ -- Michael Zanetti <michael_zanetti@gmx.net>  Sun, 01 Mar 2015 20:00:00 +0200
```

Comment:
About the name. No one is maintaining it right now, you picked it up, so as far as I'm concerned (but I don't have a say in this :stuck_out_tongue_closed_eyes: ) it's ok to use your own name, but the verdict has to be made by @mzanetti 

And I guess most of the library changes are require some (small) changes in the app/QML. So you're right about that.